resume-buddy/src/main/java/com/google/sps/servlets/ReviewerDataServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import com.google.sps.data.Reviewer;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that saves reviewer data from the form */
+@WebServlet("/reviewer-data")
+public class ReviewerDataServlet extends HttpServlet {
+
+  private Reviewer reviewer;
```

Comment:
This `Reviewer` isn't really being used right now at all. Remove it for now or add a TODO above it?

Reply:
I use reviewer to create the Gson/json in doGet()

Reply:
Oh I see it now! Sorry for not seeing it earlier.